source/usage-examples/findMany.txt

```diff
@@ -170,10 +170,10 @@ Running the preceding full examples prints the following results:
 Additional Information
 ----------------------
 
-.. TODO: Add links to references once pages are done
-
 To learn more about using builders, see :ref:`csharp-builders`.
 
+To learn how to find documents using LINQ, see :ref:`csharp-linq`.
+
 API Documentation
 ~~~~~~~~~~~~~~~~~
```

Comment:
Issue: in all of the other usage examples, the API documentation is a top level header, not nested under Additional Information. 

Suggestion: either make this header a top level header or change all the other examples to match this one.